src/Core/Extensions/Http.cs

```diff
@@ -23,6 +23,10 @@ internal static async Task ExecuteHttpProxyOperationAsync(this HttpContext conte
                 var httpClient = context.RequestServices
                     .GetService<IHttpClientFactory>()
                     .CreateClient(options?.HttpClientName ?? Helpers.HttpProxyClientName);
```

Comment:
This line will still fail, correct? :)

I think this access needs to be removed.

Reply:
Yup. 2 AM error, missed to stage this change.